builder/dockerfile/internals.go

```diff
@@ -116,7 +116,7 @@ func (b *Builder) performCopy(ctx context.Context, req dispatchRequest, inst cop
 	if inst.chownStr != "" {
 		chownComment = fmt.Sprintf("--chown=%s", inst.chownStr)
 	}
-	commentStr := fmt.Sprintf("%s %s%s in %s ", inst.cmdName, chownComment, srcHash, inst.dest)
+	commentStr := fmt.Sprintf("%s %s %s in %s ", inst.cmdName, chownComment, srcHash, inst.dest)
```

Comment:
Looking at the code, I think the intent here was to prevent adding a space if there's no `--chown` option set.

So instead, we should add a space here;

```go
chownComment = fmt.Sprintf("--chown=%s ", inst.chownStr)
```

Or perhaps even better; add a space before _and after_, 

```go
chownComment = fmt.Sprintf(" --chown=%s ", inst.chownStr)
```

and change the `commentStr` format to remove the spaces around the `%s` for the `chownComment;

```go
if inst.chownStr != "" {
	chownComment = fmt.Sprintf(" --chown=%s ", inst.chownStr)
}
commentStr := fmt.Sprintf("%s%s%s in %s ", inst.cmdName, chownComment, srcHash, inst.dest)
```

Reply:
Done